{
  try {
    return sourceAndContext.second.computeErrors(sourceAndContext.first);
  }
 catch (  AnalysisException e) {
    LOG.info(e);
  }
  return null;
}
